### PR TITLE
Solutions of qs 1 and 4

### PR DESCRIPTION
**Pull request by berasaikat**
Final solutions of qs 1 and 4 from my end

### CODE REVIEW

File: rough-work-saikat/question1.rmd [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+$\lambda$ is the sample mean. So, we will take $\lambda = 1.5$.
+
+a.  $\mathbb{P}(X \geq 1) = 1 - \mathbb{P}(X = 0) = 1 - e^{-\lambda} =
+0.78$;
+b.  $\mathbb{P}(1 \leq X < 4) = \mathbb{P}(X = 1) + \mathbb{P}(X = 2) +
+\mathbb{P}(X = 3) = e^{-\lambda} \big[\lambda + \frac{\lambda^2}{2} +
+\frac{\lambda^3}{6}\big] = 0.71$
+
+### Part 6: 
+Which probability model you would prefer over another?
```
Comment (shreyanc07) [inline rough-work-saikat/question1.rmd:124]: The reason given in this part is not clear. Please give a better clarification.

**Review by shreyanc07 — CHANGES_REQUESTED** [review]

**Review by Roudranil — APPROVED** [review]
Made a correction in line 265 of qs 4 solution. The codes run as expected.

**Comment by Roudranil** [comment]
Also @shreyanc07 's changes have been implemented.

**Review by shreyanc07 — APPROVED** [review]
All changes have been  done necessarily and code run as usual.

**Pull request merged**